app.py

```diff
@@ -155,19 +159,24 @@ def getData():
 
 	#USE CONDITIONAL ALONG WITH UI INFORMATION RECEIVED FROM THE CLIENT TO SWITCH
 	#BETWEEN HEAT MAP AND INTERPOLATION ANALYSIS
+	
+	 if heatmap == "True":
```

Comment:
This line looks like it is inset too far, and there is nothing in the conditional. I believe Python will give you an error in this case.